web/src/views/TwitterDownloaderView.vue

```diff
@@ -1,258 +1,250 @@
 <template>
-    <div class="container">
-        <div
-            class="d-flex flex-column justify-content-center
-            align-items-center">
-            <lottie-player
-                src="https://assets2.lottiefiles.com/private_files/lf30_pdS85G.json"
-                background="transparent"
-                speed="0.5"
-                style="width: 300px; height: 300px;"
-                loop
-                autoplay />
+	<div class="container">
+		<div class="d-flex flex-column justify-content-center align-items-center">
+			<div class="col-sm-12 col-md-4 text-center">
+				<div class="header-container">
+					<lottie-player
+						src="https://assets2.lottiefiles.com/private_files/lf30_pdS85G.json"
+						background="transparent"
+						class="mx-auto"
+						speed="1"
+						style="width: 15rem; height: 15rem;"
+						autoplay />
+				</div>
 
-            <div class="col-lg-6 col-md-4 text-center">
-                <figure class="text-center">
-                    <blockquote class="blockquote">
-                        <h3 class="mb-4">
-                            Twitter Video Downloader
-                        </h3>
-                    </blockquote>
+				<figure 
+					v-if="!tweetMetaData && !formError"
+					class="text-center">
+					<blockquote class="blockquote">
+						<h3 class="mb-4">
+							Twitter Video Downloader
+						</h3>
+					</blockquote>
 
-                    <figcaption class="blockquote-footer">
-                        Download Twitter Videos in seconds...
-                    </figcaption>
-                </figure>
+					<figcaption class="blockquote-footer">
+						Download Twitter Videos in seconds...
+					</figcaption>
+				</figure>
```

Comment:
Can we follow the [QR Code design](https://github.com/pizza-society/unicornify/blob/PS-59-Refactor-Web/web/src/views/QRGeneratorView.vue#L15-L22) for this section? Or you feel it look better like this?

Reply:
Yes, yours is better. I will follow it